source/photo.html

```diff
@@ -39,114 +37,93 @@ <h1 class="visually-hidden">PINK - фотоприложениe для раскр
 
       <section class="users-photos">
 
-        <h2 class="users-photos__tittle">Фотографии пользователей</h2>
+        <h2 class="users-photos__title">Фотографии пользователей</h2>
         <p class="users-photos__info">
           Взгляните на фотографии,
           которые выкладывают пользователи!
           Видите как не хватает ваших?
         </p>
 
-        <article class="users-photos__item photo-card">
-          <img src="#" alt="Фотография моста над рекой" class="photo-card__image">
-          <p class="photo-card__username">
-            Анна <span class="photo-card__online-time">(5 минут назад)</span>
+        <article class="posts">
+          <img src="#" alt="Фотография моста над рекой" class="posts__image">
+          <p class="posts__username">
+            Анна <span class="posts__online-time">(5 минут назад)</span>
           </p>
-          <p class="photo-card__comment">
+          <p class="posts__comment">
             Прыгать или нет? Напишите в
             комментарии свой совет и
             смотрите прямую трансляцию в
             перископе, задавайте свои вопросы!
           </p>
-          <div class="photo-card__likes like-block">
-            <img class="like-block__icon" src="#" alt="Иконка лайка" width="13" height="12">
-            <p class="like-block__amount">Нравится: 256</p>
-          </div>
+          <p class="posts__like">Нравится: 256</p>
         </article>
 
-        <article class="users-photos__item photo-card">
-          <img src="#" alt="Фотография камеры с машиной" class="photo-card__image">
-          <p class="photo-card__username">
-            Левон <span class="photo-card__online-time">(5 часов назад)</span>
+        <article class="posts">
```

Comment:
Множественное число было бы уместно, для именования списка, но тут нужно единственное число
```suggestion
        <article class="post">
```